tests/CodeBehind/UnitTests/BuildTests.cs

```diff
@@ -553,8 +553,15 @@ void RunTest (string testName, bool many, bool dtb, Action<TestProjectInfo, bool
 				}
 			} catch {
 				CopyLogs (testInfo, false);
+				foreach (var file in Directory.GetFiles (testInfo.OutputDirectory, "*.log", SearchOption.AllDirectories)) {
```

Comment:
There are probably `.binlog` files, too. Maybe could be:
```suggestion
				foreach (var file in Directory.GetFiles (testInfo.OutputDirectory, "*.*log", SearchOption.AllDirectories)) {
```

Reply:
Ok, yeah we are only using `*.log` files as attachments, we can wait on this.